tools/process_raw_response.py

```diff
@@ -2,7 +2,6 @@
 
 import argparse
 import json
-import datetime
```

Comment:
While this is flagged by LGTM, it is required for the `eval` on L65.

Reply:
weird, sorry for that then, datetime is back in